src/Telegram.Bot.Core/Requests/ChatIdRequestBase.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System.Diagnostics.CodeAnalysis;
+using System.Net.Http;
+using Telegram.Bot.Types;
+
+namespace Telegram.Bot.Requests
+{
+    public abstract class ChatIdRequestBase<TResponse> : RequestBase<TResponse>
```

Comment:
Why is this class needed? We tried to minimise hierarchies and used interfaces for xml comments sharing instead.

Reply:
Is there a real performance reason? Because such class allows to write less code than when implementing the interface

Reply:
It's not about performance, it's about minimizing public API space. No need to add stuff that don't bring much benefit.

If C# would have proper traits like Rust it would _probably_ make sense to separate shared properties into separate traits. In this case it's just a single class for a single property. Adding more such classes will have diminishing returns.